readme.md

```diff
@@ -137,6 +168,8 @@ $ npm -v
 
 * **Your Answer:**
 
+path is the directory that the file is located in and it comes from node.
```

Comment:
`path` is actually a node module. It has _methods_ that can give you access to a file path and/or folder.